iati/contact/models.py

```diff
@@ -1,6 +1,26 @@
 from django.db import models
-
+from home.models import AbstractContentPage
+from wagtail.admin.edit_handlers import FieldPanel
+from wagtail.core.blocks import RichTextBlock, StreamBlock, StructBlock, TextBlock
+from wagtail.core.fields import StreamField
 from wagtail.core.models import Page
 
-class ContactPage(Page):
-    pass
+
+class ContactTypeBlock(StructBlock):
+    """Model to define fields relating to a type of query."""
+    heading = TextBlock()
+    copy = RichTextBlock()
+    email = TextBlock()
+
+
+class ContactTypeStreamBlock(StreamBlock):
```

Comment:
Is this class needed - could be argued to be an unnecessary layer of abstraction...

Alternative approach, as used for the history block: https://github.com/IATI/preview-website/blob/add-about-models/iati/about/models.py#L108-L133